### PR TITLE
PORTALS-584: add news page menu component

### PR DESCRIPTION
**Pull request by jay-hodgson**
Variable number of feeds (with feed names), with optional twitter feed, or optional mailchimp signup

### CODE REVIEW

**Comment by jay-hodgson** [comment]
note that portal config will use "prod" WordPress site rss feed links (rather than the staging links found in this PR).

File: src/__tests__/lib/containers/NewsFeedMenu.test.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import * as React from 'react'
+import { shallow } from 'enzyme'
+import NewsFeedMenu, { NewsFeedMenuProps } from '../../../lib/containers/NewsFeedMenu'
+import RssFeed from '../../../lib/containers/RssFeed'
+import TwitterFeed from '../../../lib/containers/TwitterFeed'
+import MailchimpSubscribe from 'react-mailchimp-subscribe'
+
+const createShallowComponent = (props: NewsFeedMenuProps, disableLifecycleMethods: boolean = false) => {
+  const wrapper = shallow(
```
Comment (leem42) [inline src/__tests__/lib/containers/NewsFeedMenu.test.tsx]: can add type to shallow instead of cast below

Comment (leem42) [reply src/__tests__/lib/containers/NewsFeedMenu.test.tsx]: can get rid of disableLifecylceMethods

File: src/lib/containers/NewsFeedMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+import MailchimpSubscribe from 'react-mailchimp-subscribe'
+
+type MenuState = {
+  menuIndex: number
+}
+
+export type MenuConfig = {
+  feedName: string
+  feedUrl: string
+  maxItemsToShow: number
```
Comment (leem42) [inline src/lib/containers/NewsFeedMenu.tsx]: defaultItemsToShow

**Pull request merged**